Kubernetes/OrderForm/dev-orderform-kubernetes-manifest.yml

```diff
@@ -36,10 +36,21 @@ spec:
         - name: PUBLIC_BROWSE_BASE_URL
           value: "https://buyingcataloguedev.uksouth.cloudapp.azure.com"
 
+          #Organisations API base URL
+        - name: ORGANISATION_API_URL  
+          value: "http://oapi:5103"
+
           #Ordering API base URL
         - name: ORDER_API_URL 
           value: "http://orapi:5104"
 
+          #BuyingCatalogue API base URL
+        - name: SOLUTIONS_API_URL
+          valueFrom:
+                secretKeyRef:
```

Comment:
the other 3 urls are just setting the literal value, why does this one need to be a secret?

Reply:
In all honesty - it doesn't. It shouldn't be secret either, in here we're using it as a configMap. 
I went with this approach since some components that depend on bapi are already pulling the value through this secret & I wanted to be somewhat consistent (funny as it is inconsistent with the way we're pulling in other urls 😄).

Reply:
It's the approach that was there. Looking forward to deleting these files